### DISCUSSION TITLE
WALL_TEMPERATURE vs ADIABATIC SURFACE TEMPERATURE GAS

### DISCUSSION THREAD

**Discussion by ramsampath** [firemodels/fds#13786] [General]
Hello ,

We are currently using "WALL TEMPERATURE" to determine failures of equipment. For that we place a DEVC line on the point on the equipment closest  to the fire source and oriented in the direction of the source, in order to have an estimate on how quickly the equipment fails. (most conservative means - equipment temp reaches highest quickly).

 Since FDS requires DEVC should be physically placed on the object aligned with the grid,  we have run into scenarios where FDS complains that the  device is not on the surface of the equipment in the way we are creating the DEVC point. 

However there seems to another measure  "ADIABATIC SURFACE TEMPERATURE GAS"  which the manual says as follows...

"This output indicates the maximum achievable solid surface temperature at the given location XYZ that is
not actually in the vicinity of any solid surface, facing in any direction as indicated by the ORIENTATION
vector. Note that you must set the EMISSIVITY and HEAT_TRANSFER_COEFFICIENT (W/(m2·K)) on the
PROP line because there is no actual solid surface from which to infer these values."

The problem is that we don't know the HEAT_TRANSFER_COEFFICIENT since that's not part of the solid material property and dont want the user to input this. Am i correct in understanding that higher numbers indicate more heat loss at the surface so it will get lesser temperature which is probably not what we want. We want the most conservative approach in which it retains the higher temperature. 

 Would a value of 5  be quite conservative in handling different kinds of flows and practical ? What would be the lowest which would make sense ?

 From your experience, should I be using the gas temperature  surrounding the equipment instead, to get the highest temperature that equipment could be exposed to ?

Thank you for your time.

Ram.

**Comment by mcgratta** [comment]
The AST is defined as the temperature of a surface under the assumption that the conductivity of the solid material is 0; that is, it is perfectly insulated. That being said, the temperature of the (fictitious) surface would still be affected by the HTC and emissivity. Assume emissivity is one. HTC should be something typical. If the fluid is not moving, choose some typical HTC like 5 or 10 W/m2/K. If, however, you are considering something like a blow torch impinging on your surface, then you have to rethink it.

**Reply by drjfloyd** [reply]
Make sure you specify an IOR for the DEVC. If your DEVC XYZ winds up too far away from where the surface of the OBST snaps to the grid or you have multiple wall cells of different orientations near the XYZ, then FDS might not be able to indentify a unique wall cell. IOR tells FDS how to search for the wall.

The surface will see radiative and convective flux. If the gas near the surface is optically thin, then using the gas temperature could significantly underestimate the potential hazard. For example, you have a vertical surface with a fire next to the surface but not close enough for flame to touch the surface. In this case, the gas at the surface would be relatively cold, but the surface would see radiant heat from the adjacent fire. This is the benefit of the AST. It accounts for both convective and radiative heating. In optically thin cases the AST will include the effect of radiation from a fire or a hot layer above the surface. In optically thick cases the AST will wind up looking like the gas temperature near the surface. As Kevin noted, you want the HTC to be something reasonable.  If your model is being intended for use for the design of a real building, your local codes and standards may have recommendations for this.

**Comment by ramsampath** [comment]
Thank you, @mcgratta and @drjfloyd, for your insights. Based on your suggestions, I will run some sensitivity tests and report any remarkable findings.
